api/renter.go

```diff
@@ -22,7 +22,7 @@ var (
 	// contracts with if the value is not specified explicity in the call to
 	// SetSettings.
 	recommendedHosts = build.Select(build.Var{
-		Standard: uint64(30),
+		Standard: uint64(42),
```

Comment:
10 more than the default number of hosts used when uploading a file. That means much better upload speeds all the way through to the end, also means more hosts getting contracts + files overall.